aiohttp/web_runner.py

```diff
@@ -3,7 +3,7 @@
 import socket
 from abc import ABC, abstractmethod
 from pathlib import Path
-from typing import Any, List, Optional, Set, Type, Union
```

Comment:
Is this change absolutely necessary? Why wasn't it included in the original PR, then? Usually, backports should be as close as possible to the original PRs. If this is _also_ necessary on master, it should be first submitted there and backported separately.

Reply:
i had this on before the backported already. I just remove the Type because LGTM tell me it's not use